hadoop-hdds/tools/src/test/java/org/apache/hadoop/hdds/scm/cli/datanode/TestContainerBalancerSubCommand.java

```diff
@@ -206,27 +178,298 @@ public void testContainerBalancerStatusInfoSubcommandRunning()
+        "Interval between each Iteration                    0min\n" +
+        "Whether to Enable Network Topology                 false\n" +
+        "Whether to Trigger Refresh Datanode Usage Info     false\n" +
+        "Container IDs to Exclude from Balancing            None\n" +
+        "Datanodes Specified to be Balanced                 None\n" +
+        "Datanodes Excluded from Balancing                  None";
+    assertTrue(output.contains(balancerConfigOutput));
+
+    String currentIterationOutput =
+        "Current iteration info:\n" +
+        "Key                                                Value\n" +
+        "Iteration number                                   3\n" +
+        "Iteration duration                                 1h 6m 40s\n" +
+        "Iteration result                                   IN_PROGRESS\n" +
+        "Size scheduled to move                             48 GB\n" +
+        "Moved data size                                    48 GB\n" +
+        "Scheduled to move containers                       11\n" +
+        "Already moved containers                           11\n" +
+        "Failed to move containers                          0\n" +
+        "Failed to move containers by timeout               0\n" +
+        "Entered data to nodes                              \n" +
+        "80f6bc27-e6f3-493e-b1f4-25f810ad960d <- 20 GB\n" +
+        "701ca98e-aa1a-4b36-b817-e28ed634bba6 <- 28 GB\n" +
+        "Exited data from nodes                             \n" +
+        "b8b9c511-c30f-4933-8938-2f272e307070 -> 30 GB\n" +
+        "7bd99815-47e7-4015-bc61-ca6ef6dfd130 -> 18 GB";
+    assertTrue(output.contains(currentIterationOutput));
+
+    assertTrue(output.contains("Iteration history list:"));
+    String firstHistoryIterationOutput =
+        "Key                                                Value\n" +
+        "Iteration number                                   3\n" +
+        "Iteration duration                                 1h 6m 40s\n" +
+        "Iteration result                                   IN_PROGRESS\n" +
+        "Size scheduled to move                             48 GB\n" +
+        "Moved data size                                    48 GB\n" +
+        "Scheduled to move containers                       11\n" +
+        "Already moved containers                           11\n" +
+        "Failed to move containers                          0\n" +
+        "Failed to move containers by timeout               0\n" +
+        "Entered data to nodes                              \n" +
+        "80f6bc27-e6f3-493e-b1f4-25f810ad960d <- 20 GB\n" +
+        "701ca98e-aa1a-4b36-b817-e28ed634bba6 <- 28 GB\n" +
+        "Exited data from nodes                             \n" +
+        "b8b9c511-c30f-4933-8938-2f272e307070 -> 30 GB\n" +
+        "7bd99815-47e7-4015-bc61-ca6ef6dfd130 -> 18 GB";
+    assertTrue(output.contains(firstHistoryIterationOutput));
```

Comment:
Are we asserting the same thing twice? It looks like `currentIterationOutput` and `firstHistoryIterationOutput` have the same content.

Reply:
Thanks! I will fix the test data. The output between the current and historical iterations should be different.